### PR TITLE
BP-3567-Dutch translation for whole plug-in (And overall review)

### PR DESCRIPTION
**Pull request by SandervdHulst**

### CODE REVIEW

File: Model/Refund/Push.php [hunk 1]
```diff
@@ -119,7 +119,7 @@ public function __construct(
     }
 
     /**
-     * This is called when a refund is made in Buckaroo Payment Plaza.
+     * This is called when a refund is made in Buckaroo Buckaroo Plaza.
```
Comment (AlbinaBaraliu) [inline Model/Refund/Push.php:122]: maybe change this one to Buckaroo **Payment** Plaza

File: i18n/de_DE.csv [hunk 2]
```diff
@@ -158,12 +158,12 @@
 "Magento Enterprise Edition version","Magento Enterprise Edition-Version"
 "Quick reference guide","Kurzanleitung"
 "Log in to the ","Melden Sie sich an "
-"Buckaroo Payment Plaza","Buckaroo Payment Plaza"
+"Buckaroo Buckaroo Plaza","Buckaroo Buckaroo Plaza"
```
Comment (AlbinaBaraliu) [inline i18n/de_DE.csv:161]: also this one looks nicer as Buckaroo Payment Plaza ig

File: i18n/de_DE.csv [hunk 3]
```diff
@@ -158,12 +158,12 @@
...
 " to collect the following security settings:"," um die folgenden Sicherheitseinstellungen zu sammeln:"
 "Support","Die Unterstützung"
 "This extension is developed by Buckaroo.","Diese Erweiterung wird von Buckaroo entwickelt."
 "Extension basic configuration and account information","Basiskonfiguration der Erweiterung und Kontoinformationen"
-"Buckaroo Payment Plaza:","Buckaroo Payment Plaza:"
+"Buckaroo Buckaroo Plaza:","Buckaroo Buckaroo Plaza:"
```
Comment (AlbinaBaraliu) [inline i18n/de_DE.csv:166]: same, sry

File: i18n/es_ES.csv [hunk 4]
```diff
@@ -160,12 +160,12 @@
 "Magento Enterprise Edition version","Versión Edición de empresa de Magento"
 "Quick reference guide","Guía de referencia rápida"
 "Log in to the ","Iniciar sesión en"
-"Buckaroo Payment Plaza","Payment Plaza de Buckaroo"
+"Buckaroo Buckaroo Plaza","Buckaroo Plaza de Buckaroo"
```
Comment (AlbinaBaraliu) [inline i18n/es_ES.csv:163]: this one too :), sry for the comments, maybe they will be helpful to identify these typos

File: i18n/es_ES.csv [hunk 5]
```diff
@@ -160,12 +160,12 @@
...
 " to collect the following security settings:","para recopilar los siguientes ajustes de seguridad:"
 "Support","Asistencia"
 "This extension is developed by Buckaroo.","Esta extensión la ha desarrollado Buckaroo."
 "Extension basic configuration and account information","Configuración básica de la extensión e información de la cuenta"
-"Buckaroo Payment Plaza:","Payment Plaza de Buckaroo:"
+"Buckaroo Buckaroo Plaza:","Buckaroo Plaza de Buckaroo:"
```
Comment (AlbinaBaraliu) [inline i18n/es_ES.csv:168]: same

File: i18n/nl_BE.csv [hunk 6]
```diff
@@ -158,12 +158,12 @@
 "Magento Enterprise Edition version","Magento Enterprise Edition versie"
 "Quick reference guide","Eerste stappen"
 "Log in to the ","Log in bij "
-"Buckaroo Payment Plaza","Buckaroo Payment Plaza"
+"Buckaroo Buckaroo Plaza","Buckaroo Buckaroo Plaza"
```
Comment (AlbinaBaraliu) [inline i18n/nl_BE.csv:161]: same

File: i18n/nl_BE.csv [hunk 7]
```diff
@@ -158,12 +158,12 @@
...
 " to collect the following security settings:"," om de volgende beveiligingsgegevens te verzamelen:"
 "Support","Support"
 "This extension is developed by Buckaroo.","Deze extensie is ontwikkeld door Buckaroo."
 "Extension basic configuration and account information","Algemene instellingen en accountgegevens"
-"Buckaroo Payment Plaza:","Buckaroo Payment Plaza:"
+"Buckaroo Buckaroo Plaza:","Buckaroo Buckaroo Plaza:"
```
Comment (AlbinaBaraliu) [inline i18n/nl_BE.csv:166]: same

File: i18n/nl_BE.csv [hunk 8]
```diff
@@ -300,7 +300,7 @@
...
+"Due to compound rounding errors in Magento when using currencies other than the store's base currency when creating refunds, the amount that you refund may be several cents more or less than you intended. If you wish to make sure that the correct amount is refunded, please create an offline refund and then  use Buckaroo's Buckaroo Plaza to do the actual refund.","Wegens oplopende afrondingsfouten in Magento wanneer gekrediteerd wordt in een andere valuta dan de winkel's basisvaluta, kan het voorkomen dat er enkele centen verschil zitten tussen het bedrag dat gekrediteerd moet worden en het bedrag dat daadwerkelijk teruggestort wordt. Indien u zeker wilt zijn dat het juiste bedrag teruggestort wordt, kunt u het beste een offline refund aanmaken en het eigenlijke bedrag terugstorten vanuit Buckaroo's Buckaroo Plaza."
```
Comment (AlbinaBaraliu) [inline i18n/nl_BE.csv:303]: same

File: i18n/nl_BE.csv [hunk 9]
```diff
@@ -362,8 +362,8 @@
...
+"Cannot Refund via Magento Backend. Partial refunds combined with a payment fee can only be refunded via the Buckaroo Buckaroo Plaza, see also the <a href="https://support.buckaroo.nl" target="_blank">KB article</a>.<br><a href="https://plaza.buckaroo.nl" target="_blank">Open a new window to the Buckaroo Buckaroo Plaza</a>.","Deze refund is niet mogelijk. Gedeeltelijke refunds gecombineerd met de payment fee kunnen alleen worden teruggestort via de Buckaroo Buckaroo Plaza, zie ook het <a href="https://support.buckaroo.nl" target="_blank">KB artikel</a>.<br><a href="https://plaza.buckaroo.nl" target="_blank">Open een nieuw venster om naar de Buckaroo Buckaroo Plaza</a>."
```
Comment (AlbinaBaraliu) [inline i18n/nl_BE.csv:365]: same

File: i18n/nl_BE.csv [hunk 10]
```diff
@@ -362,8 +362,8 @@
...
+"Cannot refund this order via Magento Backend for now, we are working on a solution! Credit Management orders can only be refunded via the Buckaroo Buckaroo Plaza.<br><a href="https://plaza.buckaroo.nl" target="_blank">Open a new window to the Buckaroo Buckaroo Plaza</a>.",,"Deze refund is momenteel niet mogelijk, wij werken aan een oplossing! Credit Management refunds kunnen alleen worden teruggestort via de Buckaroo Buckaroo Plaza.<br><a href="https://plaza.buckaroo.nl" target="_blank">Open een nieuw venster om naar de Buckaroo Buckaroo Plaza</a>."
```
Comment (AlbinaBaraliu) [inline i18n/nl_BE.csv:366]: same

File: view/adminhtml/templates/sales/order/creditmemo/create/rounding_warning.phtml [hunk 11]
```diff
@@ -21,5 +21,5 @@
  */
 ?>
 <div class="message message-warning warning buckaroo-magento2-warning-message">
-    <strong><?php echo __("Warning:") ?> </strong><?php echo __("Due to compound rounding errors in Magento when using currencies other than the store's base currency when creating refunds, the amount that you refund may be several cents more or less than you intended. If you wish to make sure that the correct amount is refunded, please create an offline refund and then  use Buckaroo's payment plaza to do the actual refund.") ?>
+    <strong><?php echo __("Warning:") ?> </strong><?php echo __("Due to compound rounding errors in Magento when using currencies other than the store's base currency when creating refunds, the amount that you refund may be several cents more or less than you intended. If you wish to make sure that the correct amount is refunded, please create an offline refund and then  use Buckaroo's Buckaroo Plaza to do the actual refund.") ?>
```
Comment (AlbinaBaraliu) [inline view/adminhtml/templates/sales/order/creditmemo/create/rounding_warning.phtml:24]: same

File: Model/Refund/Push.php (hunk 1 above)
Comment (Buckaroo-Rene) [reply Model/Refund/Push.php]: **This should be Buckaroo Plaza:**
 * This is called when a refund is made in the Buckaroo Plaza.

File: i18n/de_DE.csv (hunk 2 above)
Comment (Buckaroo-Rene) [reply i18n/de_DE.csv]: **This should be Buckaroo Plaza:**
"Buckaroo Plaza","Buckaroo Plaza"

File: i18n/de_DE.csv (hunk 3 above)
Comment (Buckaroo-Rene) [reply i18n/de_DE.csv]: **This should be Buckaroo Plaza:**
"Buckaroo Plaza:","Buckaroo Plaza:"

File: i18n/es_ES.csv (hunk 5 above)
Comment (Buckaroo-Rene) [reply i18n/es_ES.csv]: **This should be:**
"Buckaroo Plaza:","Buckaroo Plaza:"

File: i18n/nl_BE.csv (hunk 7 above)
Comment (Buckaroo-Rene) [reply i18n/nl_BE.csv]: **This should be:**
"Buckaroo Plaza:","Buckaroo Plaza:"

**Review by Buckaroo-Rene — APPROVED** [review]

**Pull request merged**